public/static/js/spawned_hosts.js

```diff
@@ -495,6 +495,7 @@ mciModule.controller('SpawnedHostsCtrl', ['$scope', '$window', '$timeout', '$q',
     $scope.setVolume = function (volume) {
       if (volume) {
         $scope.homeVolumeID = volume.volume_id;
+        $scope.homeVolumeDisplay = volume.display_name ? volume.display_name : volume.volume_id;
```

Comment:
This logic can go in the HTML template (you don't need to have this in $scope)

Reply:
But if this logic goes into the html template it's really awkward I think, because it's like:
```
if display != "" { use display }
else if id != "" { use id }
else { use "New Volume" }
```
I think breaking it up is cleaner.